src/test/java/nextstep/subway/station/StationAcceptanceTest.java

```diff
@@ -111,5 +117,22 @@ void getStations() {
     @DisplayName("지하철역을 제거한다.")
     @Test
     void deleteStation() {
+        // given
+        int id1 = createNewStation("강남역").body().jsonPath().get("id");
+        int id2 = createNewStation("역삼역").body().jsonPath().get("id");
```

Comment:
```suggestion
        int 강남역ID = createNewStation("강남역").body().jsonPath().get("id");
        int 역삼역ID = createNewStation("역삼역").body().jsonPath().get("id");
```

좀 더 직관적인 이름으로 가독성을 향상시킬수 있겠네요!